crates/cli/src/config.rs

```diff
@@ -850,6 +850,13 @@ impl Config {
         }
     }
 
+    pub fn new_with_localhost() -> Self {
+        Self {
+            home: RawConfig::default(),
+            proj: RawConfig::new_with_localhost(),
```

Comment:
```suggestion
            home: RawConfig::new_with_localhost(),
            proj: RawConfig::default(),
```

Project configs are essentially unsupported at this point, and probably don't work.